utils/utils.py

```diff
@@ -71,8 +73,8 @@ def load_from_checkpoint(checkpoint, model, optimizer=None, inference:str=''):
     strict_loading = False if not inference else True
     model.load_state_dict(checkpoint['model'], strict=strict_loading)
     logging.info(f"=> loaded model\n")
-    if optimizer and 'optimizer' in checkpoint.keys():    # 2nd condition if loading a model without optimizer
-        optimizer.load_state_dict(checkpoint['optimizer'], strict=False)
+    # if optimizer and 'optimizer' in checkpoint.keys():    # 2nd condition if loading a model without optimizer
```

Comment:
Can we delete it ?

Reply:
I think this should stay (although feel free to disagree)

- as if we keep it:
     if `optimizer == None` then it will not change anything
     if `optimizer != None` it will load a better/more accurate/more pre-trained checkpoint 
                                                               ^ for lack of better words
- if we remove it
     we should remove the optimizer var from the params, docs, & return line
                                                                               (the PR's Lines 55, 60, 78)

@victorlazio109 I was having issues with this line too & the way I was able to solve it was to remove the `strict=False` from line 77

my code:
```
    if optimizer and 'optimizer' in checkpoint.keys():    # 2nd condition if loading a model without optimizer
        optimizer.load_state_dict(checkpoint['optimizer'])#, strict=False)
```

- lastly, if it is removed to be able to load checkpoints with diff optimizers..
maybe we could add an `if` statement?
here is a helpful link if this is the case:
https://stackoverflow.com/questions/64301566/how-to-check-if-an-object-is-a-certain-pytorch-optimizer

Reply:
oh nvm, you commented out strict=False, I will try